ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -1,6 +1,7 @@
 /* Copyright 2016 Endless Mobile, Inc. */
 
 #include "eknc-query-object.h"
+#include "eknc-query-object-private.h"
```

Comment:
The usual pattern is to include the private header, which will then include the public one. This avoids having a header that requires a precise ordering of inclusion.